dotnet/samples/KernelSyntaxExamples/Example21_OpenAIPlugins.cs

```diff
@@ -47,12 +47,14 @@ public static async Task CallKlarnaAsync()
 
         var plugin = await kernel.ImportPluginFromOpenAIAsync("Klarna", new Uri("https://www.klarna.com/.well-known/ai-plugin.json"));
 
-        var arguments = new KernelArguments();
-        arguments["q"] = "Laptop";      // Category or product that needs to be searched for.
-        arguments["size"] = "3";        // Number of products to return
-        arguments["budget"] = "200";    // Maximum price of the matching product in local currency
-        arguments["countryCode"] = "US";// ISO 3166 country code with 2 characters based on the user location.
-                                        // Currently, only US, GB, DE, SE and DK are supported.
+        var arguments = new KernelArguments
+        {
+            ["q"] = "Laptop",      // Category or product that needs to be searched for.
+            ["size"] = "3",        // Number of products to return
+            ["budget"] = "200",    // Maximum price of the matching product in local currency
+            ["countryCode"] = "US" // ISO 3166 country code with 2 characters based on the user location.
+        };
+        // Currently, only US, GB, DE, SE and DK are supported.
```

Comment:
One goal of the kernel syntax example sis developer sand copy/paste snippets of code into their applications. Will the majority of developers already be using C# 12?

Reply:
I don't have data on that, unfortunately. The repo was already using C# 10, which is later than anyone using NET Framework targets by default (by default it uses C# 7.3 unless overridden). But anyone can opt-in to a newer language version, regardless of target framework, so it's generally not an imposition to move to a newer language version, and features in these newer releases are natural fits for the kinds of workloads SK supports, eg raw string literals for working with JSON, or collection expression syntax being very natural for someone familiar with python, etc. Your concern is totally fair, but I don't think it should stop us from making forward, simplifying progress here.